app/scripts/controllers/visualizer.js

```diff
@@ -195,6 +195,21 @@ angular.module('openhimConsoleApp')
       }
     };
 
+    // function to add new visualizer to list
+    $scope.addVisualiser = function addVisualiser(){
+
+      // open visualizer settings modal 
+      $modal.open({
+        templateUrl: 'views/visualizerModal.html',
+        controller: 'VisualizerModalCtrl',
+        resolve: {
+          settingsStore: function () { return null; },
```

Comment:
I left it there to be used while editing to pass the selected visualizer data through to the modal. Are you doing that some other way?